compiler-rt/lib/msan/msan_flags.inc

```diff
@@ -25,6 +25,11 @@ MSAN_FLAG(bool, poison_stack_with_zeroes, false, "")
 MSAN_FLAG(bool, poison_in_malloc, true, "")
 MSAN_FLAG(bool, poison_in_free, true, "")
 MSAN_FLAG(bool, poison_in_dtor, true, "")
+MSAN_FLAG(bool, print_faulting_instruction, false,
```

Comment:
What is the theory here? Why would I want to have one (compile) but not the other (runtime) - only to get warned about it?

Reply:
We envision an Msan Compiler-rt Universe of runtime options:
- this patch: `print_faulting_instruction`
- future work?: `ignore_faulting_instruction` (for example, skip over all Vector Convert Half-word Unsigned to Long Unsigned [VCVTHULU] instructions)
- etc.

A user will be able to pick and choose any of these options at runtime, assuming the binary is already compiled with the embedded instruction.

Reply:
Ok.